_authors/anandajoti.md

```diff
@@ -1,3 +1,6 @@
 ---
 title: "Bhikkhu Ānandajoti"
 ---
+Bhikkhu Anandajoti was born near Birmingham in England. Godwin Samararatne was his first Dhamma teacher and his first Buddhist meditation teacher. Anandajoti ordained in the Theravāda tradition in 1995 and received higher ordination the following year. He is currently a resident in Vivekavana Solitude Grove, a remote meditation centre near the top of a mountain in Penang State, Malaysia.
```

Comment:
`his first Dhamma teacher and his first Buddhist meditation teacher` is a bit redundant. Perhaps just, `his first meditation teacher`?

And `resident in` should be `resident of`